whitebox/tests/v1/mock_data.py

```diff
@@ -73,6 +73,22 @@
     probability="n/a",
 )
 
+model_regression_create_payload = dict(
+    name="Regression Model",
+    description="Regression Model description",
+    type=ModelType.regression,
+    features={
+        "feature1": "numerical",
```

Comment:
Why you didn't use the load_diabetes dataset? The mock data you created is not a representative example of a regression. Usually in the regression we find float numbers in the target column (e.g. stock price, temperature etc.) and we DON'T have labels!

Reply:
I used load_diabetes but not to create the model! 😂

Reply:
Plus this shows that the labels param should be optional!

Reply:
Actually there are no labels in regression (it's a bit confusing term). In any case if it is ok, no problem from my side!